app/src/main/res/drawable/ic_caution.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="361.117dp"
```

Comment:
It should not be like this, as primary and secondary colours can change. Instead of hardcoding white colour, implement a `tint`

Reply:
Updated.

Reply:
Where?

Reply:
After that
```
android:height="361.117dp"
    android:viewportWidth="361.117"
    android:viewportHeight="361.117"
    android:tint="@color/white">
```

Reply:
It should be the secondary color, and not white

Reply:
@abishekvashok 
Should I change it like that, same color in tint and fill color?
``` xml
<vector
    android:tint="#ffffff">
    <path
        android:fillColor="#ffffff"
        />
    <path
        android:fillColor="#ffffff"
         />
    <path
        android:fillColor="#ffffff"
         />
</vector>
```

Reply:
Please use a color matching with a secondary color.

Reply:
Updated